<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>535</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>365</width>
    <height>535</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>365</width>
    <height>535</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TheTimeApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Будильник</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>331</width>
        <height>441</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTimeEdit" name="alarm_time">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="alarm_set">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Список установленных будильников</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="alarm_list_widget">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>13</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="alarm_delete_button">
           <property name="text">
            <string>Удалить выбранный</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="alarm_clearlist_button">
           <property name="text">
            <string>Очистить список</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Секундомер</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>321</width>
        <height>431</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,5,0,5,0,5,4">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="stopw_timer_label">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Время</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>70</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopw_start_button">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Старт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopw_pause_button">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Пауза</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopw_reset_button">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Сброс</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopw_round_button">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Круг</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopw_save_button">
         <property name="text">
          <string>Сохранить запись</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="stopw_rounds_list_widget">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Таймер</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>321</width>
        <height>431</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>30</number>
         </property>
         <item row="2" column="0">
          <widget class="QSpinBox" name="cdown_hours">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="cdown_preset_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="cdown_hours_label">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Часы</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="cdown_minutes">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>59</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="cdown_minutes_label">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Минуты</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="cdown_seconds_label">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Секунды</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="cdown_seconds">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>59</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="cdown_preset_1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="cdown_preset_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="cdown_reset_button">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Сброс</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="cdown_time_lcd">
         <property name="digitCount">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cdown_start_button">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Старт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cdown_pause_button">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Пауза</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>365</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
